tests/simulator/test_vtk_diagnostics.py

```
@@ -0,0 +1,209 @@
+
+    L = 0.5
+
+    def density(x, y):
+        Ly = sim.simulation_domain()[1]
+        return (
+            0.4
+            + 1.0 / np.cosh((y - Ly * 0.3) / L) ** 2
+            + 1.0 / np.cosh((y - Ly * 0.7) / L) ** 2
+        )
+
+    def S(y, y0, l):
```

Ruff (0.14.14)

[error] Ambiguous variable name: `l`

(E741)